packages/hooks/src/useSigner.ts

```diff
@@ -1,18 +0,0 @@
-import { Signer, TypedDataSigner } from '@ethersproject/abstract-signer'
-import { useWeb3React } from '@web3-react/core'
-import { useMemo } from 'react'
-
-/**
- * Hook returning the current signer logged in to the website. This signer can and should
- * be used to sign messages or transactions
- * @returns (Signer & TypedDataSigner) | undefined
- */
-export default function useSigner(): (Signer & TypedDataSigner) | undefined {
```

Comment:
This is moved to the application so they can decide the way to retrieve their signer